tutorials/create-cloud-build-image-factory-using-packer/index.md

```diff
@@ -0,0 +1,164 @@
+gcloud config set account $ACCOUNT
+```
+
+## Task 1  Set the project variable (Skip this step if you created a new project above)
+Ensure you are working with the project you want to use in gcloud.  
+For more information on configuraitons see [configurations](https://cloud.google.com/sdk/gcloud/reference/config/configurations/) 
+```sh
+gcloud config configurations activate $MY_CONFIGURATION #The configuration for the project you want to use
+PROJECT=$(gcloud config get-value project)
+```
+
+## Task 2  Enable the required services
+```sh
+gcloud services enable sourcerepo.googleapis.com
+gcloud services enable cloudapis.googleapis.com 
+gcloud services enable compute.googleapis.com
+gcloud services enable servicemanagement.googleapis.com 
+gcloud services enable storage-api.googleapis.com  
+gcloud services enable cloudbuild.googleapis.com
+```
+
+## Task 3 Give the Cloud Build user Project Editor permissions 
+```sh
+CLOUD_BUILD_ACCOUNT=$(gcloud projects get-iam-policy $PROJECT --filter="(bindings.role:roles/cloudbuild)"  --flatten="bindings[].members" --format="value(bindings.members[])")
+
+gcloud projects add-iam-policy-binding $PROJECT \
+  --member $CLOUD_BUILD_ACCOUNT \
+  --role roles/editor   
+```
+
+## Task 4 Create the Cloud Source Repository for your image creator 
+```sh
+gcloud source repos create helloworld-image-factory
+```
+
+## Task 5 Create the build triggers for the image creator source repository 
+Go to to the [build triggers page](https://console.cloud.google.com/cloud-build/triggers) and create a trigger.  
+
+1. Click "Create Trigger" 
+1. Select "Cloud Source Repository" Click "Continue".
+1. Select "helloworld-image-factory" anc click "Continue" 
+1. Enter "Hello world image factory" for Name." 
+1. Set the trigger for "Tag". 
+1. Set the build type to "cloudbuild.yaml"
+1. Set the substitution, ``_IMAGE_FAMILY`` to centos-7
+1. Set the substitution, ``_IMAGE_ZONE`` to the zone you want to use the value of ```$ZONE```. 
+1. Click "Create Trigger" 
+
+**Note: To see a list of image families:** 
+```sh
+gcloud compute images list | awk '{print $3}'  | awk '!a[$0]++'
+```
+
+## Task 6 Add the packer Cloud Build image to your project
+
+```sh
+project_dir=$(pwd)
+cd /tmp
+git clone https://github.com/GoogleCloudPlatform/cloud-builders-community.git
+cd cloud-builders-community/packer
+gcloud builds submit --config cloudbuild.yaml
+rm -rf /tmp/cloud-builders-community
+cd $project_dir
+```
+
+## Task 7 Add your helloworld-image-factory google repository as a remote repository with the name 'google'
+1. (Only if not running in Cloud Shell) setup your google credentials for git. 
+```sh 
+gcloud init && git config --global credential.https://source.developers.google.com.helper gcloud.sh
+```
+2. Add the google cloud repo as a remote. 
+```sh 
+git remote add google \
+  https://source.developers.google.com/p/$PROJECT/r/helloworld-image-factory
+```
+
+## Task 8 Push the repository and tags to google
```

Comment:
Even if you don't take my earlier suggestion to combine these tasks into a simple list of procedures, your task 8 seems so intimately related to task 7 that I think they should be combined.